test/python/visualization/test_gate_map.py

```diff
@@ -139,7 +258,295 @@ def test_plot_error_map_backend_v2(self):
     @unittest.skipUnless(optionals.HAS_SEABORN, "Seaborn not installed")
     def test_plot_error_map_over_100_qubit(self):
         """Test plotting error map with large fake backend."""
-        backend = Fake127QPulseV1()
+        coupling_map = [
```

Comment:
Can be left for a follow-up but we could maybe take advantage of some of the pre-stored coupling maps in `test` not to have to hard-code it here. Visual tests are annoying to change so I am ok with this as an intermediate step.

Reply:
I added `KYOTO_CMAP` (127 qubits) in 36633d17c5db1e8718ac069e35fe4c3fe22b8125